core/templates/dev/head/services/contextual/UrlService.js

```diff
@@ -34,10 +38,28 @@ oppia.factory('UrlService', ['$window', function($window) {
       return urlParts[1] === 'embed';
     },
     getPathname: function() {
-      return $window.location.pathname;
+      return this.getCurrentUrl().pathname;
+    },
+    getParamValuesAsList: function(queryTag) {
+      var url = this.getCurrentUrl().href.slice(
+        this.getCurrentUrl().href.indexOf('?') + 1).split('&');
+      var parentIdArray = [];
+      for (var i = 0; i < url.length; i++) {
+        var urlparam = url[i].split('=');
+        if (urlparam[0] === queryTag) {
+          parentIdArray.push(urlparam[1]);
+        }
+      }
+      return parentIdArray.length > 0 ? parentIdArray : null;
```

Comment:
In the null case this should return an empty array.

Reply:
Done